docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -1607,6 +1611,52 @@ xrdsst csr delete --ss <SECURITY_SERVER_NAME> --key <KEY_ID> --csr <CSR_ID>
 * <CSR_ID> id of the CSR to be deleted, multiple values can also be given, separated by comma, e.g. D9C0A62D8F67BD2A64B9BE26CDAF3064DDE547DE,6234EAA48BDEF552A1EBC88C4797E147024975ED
 
 
+#### 4.2.16 Security Server management
+##### 4.2.16.1 List Security Servers
+
+* Access rights: Any role
+
+Listing security servers can be done with:
+```
+xrdsst security-server list
+```
+This command will display the union of all the security servers discover by each security server stored in the configuration file.
```

Comment:
"all security servers discovered by security server"? Seems confusing. Better to state something like "This command will display all discovered security servers"